runtime-android-x86_64/Dockerfile

```
@@ -67,20 +66,16 @@ RUN chmod +x /tmp/vncpass.sh; sync && \
     rm /tmp/vncpass.sh && \
     apt-get remove -y expect && apt-get autoremove -y
 
-# Setup systemd services
+# Setup supervisor services
+RUN apt-get install -y supervisor=4.1.0-*
```

Either put this with the other installs on line 55, or it should look like this
```suggestion
RUN apt-get update -y && apt-get install -y supervisor=4.1.0-*
```
there's a highly likely chance it'll usually be fine without the update, but in case of cached previous layers this layer _could_ fail in the future.